src/mixs/schema/deprecated.yaml

```diff
@@ -0,0 +1,20 @@
+name: GSC-Deprecated-Elements
+title: Genomic Standards Consortium Deprecated Elements
+description: Elements that have been deprecated over time, in the schema for the Genomic Standards Consortium (GSC)
+id: https://w3id.org/mixs/deprecated
```

Comment:
From @sujaypatil96 


"LinkML doesn't do "link checking" of the URLs in id slots or wherever else so technically you would be okay putting in a non-resolvable URL like here: https://github.com/microbiomedata/berkeley-schema-fy24/blob/main/src/schema/deprecated.yaml#L1, although it is highly discouraged. But yes, I would recommend adding something like: https://w3id.org/mixs/deprecated because we can configure [w3id resolvers](https://github.com/perma-id/w3id.org/tree/master/mixs) on anything with https://w3id.org/mixs/ as a base URL."

I feel like here you're saying don't make a fake URL, but you can... so I did? Should I put in an issue to make this a real URL??

Reply:
Sure! an issue to make the URL resolvable would be great. It would need to be fixed with a PR on the .htaccess file in w3id repo's mixs folder.

But yes, it's fine to make a URL like `https://w3id.org/mixs/deprecated` and have that on the `id` key in the `deprecated.yaml` file.